### PR TITLE
Add var uwsgi_workers to serviceanalyzer

### PR DESCRIPTION
**Pull request by patsevanton**

### CODE REVIEW

File: reportportal/templates/analyzer-statefulset.yaml [hunk 1]
```diff
@@ -58,6 +58,8 @@ spec:
           value: "analyzer-default"
         - name: ES_HOSTS
           value: "{{ .Values.elasticsearch.endpoint }}"
+        - name: UWSGI_WORKERS
```
Comment (HardNorth) [inline reportportal/templates/analyzer-statefulset.yaml:61]: Please add a condition for those who do not have this value in `values.yaml`, as here: https://github.com/reportportal/kubernetes/blob/master/reportportal/templates/analyzer-statefulset.yaml#L47

**Review by HardNorth — CHANGES_REQUESTED** [review]

File: reportportal/templates/analyzer-statefulset.yaml (hunk 1 above)
Comment (patsevanton) [reply reportportal/templates/analyzer-statefulset.yaml:62]: I added check of Values.serviceanalyzer.uwsgi_workers

File: reportportal/templates/analyzer-statefulset.yaml [hunk 2]
```diff
@@ -58,6 +58,10 @@ spec:
           value: "analyzer-default"
         - name: ES_HOSTS
           value: "{{ .Values.elasticsearch.endpoint }}"
+        {{ if .Values.serviceanalyzer.uwsgi_workers }}
```
Comment (HardNorth) [inline reportportal/templates/analyzer-statefulset.yaml:61]: Please start the condition as: `{{- `

File: reportportal/templates/analyzer-statefulset.yaml [hunk 3]
```diff
@@ -58,6 +58,10 @@ spec:
...
+        - name: UWSGI_WORKERS
+          value: "{{ .Values.serviceanalyzer.uwsgi_workers }}"
+        {{ else }}
```
Comment (HardNorth) [inline reportportal/templates/analyzer-statefulset.yaml:64]: `{{- end }}`

File: reportportal/values.yaml [hunk 4]
```diff
@@ -130,6 +130,7 @@ serviceanalyzer:
   tag: 5.5.0
   name: analyzer
   pullPolicy: Always
+  uwsgi_workers: 2
```
Comment (HardNorth) [inline reportportal/values.yaml:133]: We use lowerCamelCase style for values.

**Review by HardNorth — CHANGES_REQUESTED** [review]

**Comment by n0rm4l-real** [comment]
Also got OOM with default values.
Any plans to merge this PR?

File: reportportal/templates/analyzer-statefulset.yaml [hunk 5]
```diff
@@ -58,6 +58,10 @@ spec:
           value: "analyzer-default"
         - name: ES_HOSTS
           value: "{{ .Values.elasticsearch.endpoint }}"
+        {{- if .Values.serviceanalyzer.uwsgiWorkers }}
+        - name: UWSGI_WORKERS
+          value: "{{ .Values.serviceanalyzer.uwsgiWorkers }}"
+        {{- else }}
```
Comment (HardNorth) [inline reportportal/templates/analyzer-statefulset.yaml:64]: Please pay attention you specified `{{- else }}` when there is no "else" condition, you should end this if with `{{- end }}`.

**Review by HardNorth — CHANGES_REQUESTED** [review]

**Comment by HardNorth** [comment]
> Also got OOM with default values.
> Any plans to merge this PR?

As soon as it get fixed and tested.

File: reportportal/templates/analyzer-statefulset.yaml (hunk 5 above)
Comment (patsevanton) [reply reportportal/templates/analyzer-statefulset.yaml]: I'm inconsiderate

**Comment by patsevanton** [comment]
Can i help for merge this pull request?

**Comment by HardNorth** [comment]
@patsevanton I need to test it first, but I was busy with other things last weeks.

**Review by HardNorth — APPROVED** [review]

**Comment by HardNorth** [comment]
@patsevanton Thank you!

**Pull request merged**